src/MGroup.FEM/Interpolation/IsoparametricInterpolation3DBase.cs

```diff
@@ -29,7 +29,7 @@ public IsoparametricInterpolation3DBase(int numFunctions)
 		/// <summary>
 		/// See <see cref="IIsoparametricInterpolation3D.CellType"/>.
 		/// </summary>
-		public CellType CellType { get; }
+		public virtual CellType CellType { get; }
```

Comment:
Why is this virtual?

Reply:
Fix of bug with continuum element cell type that returns. This is already fixed in continuum 2D by Serafeim.